lib/release_notes_bot/persists.ex

```diff
@@ -53,8 +53,12 @@ defmodule ReleaseNotesBot.Persists do
              ),
              ReleaseNotesBot.Finch
            ) do
-        {:ok, _response} ->
-          {:ok, endpoint_source}
+        {:ok, response} ->
+          {:ok, load} = Jason.decode(response.body)
+
+          {:ok,
+           endpoint_source <>
+             "#{load["id"]}/#{title |> replace_spaces_with_plus_signs |> drop_question_mark}"}
```

Comment:
i'm asking out of ignorance, what's the benefit of making these very special functions instead of doing something more generic like `format`?

Reply:
This is a good idea - calling the function `format_confluence` or something like that. As more persistence providers (like google drive, microsoft onedrive, etc) are added, I think this can be added in then.